CHANGELOG.md

```diff
@@ -4,6 +4,26 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.4.3] - TBD
+
+### Added
+- Sharded Grad Scaler works with cpu offload in mixed and full precision.
+- Chore: `Pre-Commit` will be the de-facto standard to run all linter checks (black, isort, mypy, flake8 etc.)
+on all commits locally and in CI. [#849]
+
+### Changed
+- Cleanup: Moving forward we would be testing all of our code with Python 3.9.7, CUDA 11.2 and the following three versions of PyTorch [#847]:
+  - the most recent stable version
```

Comment:
Have we added the current versions we are supporting? I don't see it in the README. We should also update the [installation requirements](https://github.com/facebookresearch/fairscale/blob/main/docs/source/installation_instructions.rst)

Reply:
I have updated the torch version in the installation requirements file.

Reply:
the versions have been added in the readme